src/main/java/racing/Cars.java

```diff
@@ -15,4 +17,18 @@ public Cars(List<Car> cars) {
     public Iterator<Car> iterator() {
         return cars.iterator();
     }
+
+    public List<Car> getWinnerList() {
```

Comment:
우승자 목록도 하나의 객체로 도출하면 어떨까요?
```suggestion
    public Winners getWinners() {
```